src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/WasmTestCommandArguments.cs

```diff
@@ -10,6 +10,28 @@ namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm
 {
     internal class WasmTestCommandArguments : TestCommandArguments
```

Comment:
You could create a new override of `Validate()` in this class and validate the args here. It should also call `base.Validate()` too